web_src/less/_repository.less

```diff
@@ -2782,31 +2795,43 @@ tbody.commit-list {
   text-align: left;
 }
 
-@media @mediaSm {
-  .ui.stackable.menu {
-    &.mobile--margin-between-items > .item {
-      margin-top: 5px;
-      margin-bottom: 5px;
-    }
+.ui.secondary.menu {
+  .item > .input,
+  .item > .button {
+    margin: 0;
+  }
 
-    &.mobile--no-negative-margins {
-      margin-left: 0;
-      margin-right: 0;
+  &.flex-wrappable {
+    margin-bottom: 0;
```

Comment:
Why not use the `mb-0` helper instead of adding this class? Also `mb-1` for `.item`.

Reply:
This would be a helper class like those it replaces: it applies recustively so I don't have to add `mb-1` to every single `.item` in case I forget one. Didn't you create those classes (forgive me if I'm wrong)?

Reply:
Yeah I guess that usage is fine. Yes it was me who added the helpers.